config/urls.py

```diff
@@ -1,8 +1,14 @@
 """Project URL Configuration"""
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
```

Comment:
Nice, I didn't know that you could export setting in such a nice way :)

Reply:
[Django docs](https://docs.djangoproject.com/en/2.1/howto/static-files/) brought me here.